ruby/Rakefile

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+require "bundler/gem_tasks"
+require "rspec/core/rake_task"
+require "standard/rake" unless RUBY_VERSION < "2.5"
+
+require "fileutils"
+require "http"
+require "pry"
+require "rubygems/package"
+
+RSpec::Core::RakeTask.new(:spec)
+
+LIB_GITHUB_RELEASES = {
+  "0.2.0" => [
+    {
+      file: "libddprof-x86_64-unknown-linux-gnu.tar.gz",
+      sha256: "cba0f24074d44781d7252b912faff50d330957e84a8f40a172a8138e81001f27",
+      ruby_platform: "x86_64-linux"
+    },
+    {
+      file: "libddprof-x86_64-alpine-linux-musl.tar.gz",
+      sha256: "d519a6241d78260522624b8e79e98502510f11d5d9551f5f80fc1134e95fa146",
+      ruby_platform: "x86_64-linux-musl"
+    }
+  ]
+  # Add more versions here
+}
+
+task default: [
+  :spec,
+  (:'standard:fix' unless RUBY_VERSION < "2.5")
+].compact
+
+desc "Download lib release from github"
+task :fetch do
+  Helpers.each_github_release_variant do |file:, sha256:, target_directory:, target_file:, **_|
+    target_url = "https://github.com/DataDog/libddprof/releases/download/v#{Libddprof::LIB_VERSION}/#{file}"
+
+    if File.exist?(target_file)
+      if Digest::SHA256.hexdigest(File.read(target_file)) == sha256
+        puts "Found #{target_file} matching the expected sha256, skipping download"
+        next
+      else
+        puts "Found #{target_file} BUT IT DID NOT MATCH THE EXPECTED sha256, skipping download"
```

Comment:
I've added printing of the hash in 8d23e241cbec24587ff0fddcc14f0e9bda61b63a .

> Also, does this really skip the download? There's no next so it would fall through and proceed, correct?

Oops, that was just too much copy pasta. The intention is to just redownload the file when the hash doesn't match, so the code was correct, but the logging was misleading. Fixed in the same commit as above.